packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -0,0 +1,71 @@
+/* stylelint-disable  */
+
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+%terra-demographics-banner {
+  background: $demographics-background;
+  color: #fff;
+  display: -ms-flex;
```

Comment:
We've been working on the idea the consumers will have [autoprefixer configured](https://github.com/cerner/terra-core/blob/master/packages/terra-site/webpack.config.js#L49-L57) in their build tools. With this, I believe autoprefixer will handle adding this.